charts/argo-workflows/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: argo-workflows
```

Comment:
Just out of curiosity; you are not defining an appVersion and set the tag right away in the values.yaml. Does this have a specific reason?

Reply:
The main motivation (or why I am personally always doing it) is to support automated image tag updates in the future, see https://github.com/argoproj/argo-helm/issues/536. If all container images are set consistently from within the values.yaml, then updates are possible out-of-the-box (at least for renovatebot should we ever consider using it).

It might make sense to set `appVersion: "v3"`?

Reply:
I see that tools like renovatebot might have trouble with the appVersion; but I believe it would be "the other way around" to now for this chart to not adopt the industry-standard practices, instead of finding a way for the other tools to support the appVersion attribute.

I would propose to use the helm-standard "appVersion" to contain the argo version and let image.tag default to .Chart.AppVersion

Reply:
I'm not completely convinced yet, but since my use case is at this point entirely hypothetical I have reverted back to using appVersion.